rampwf/score_types/detection.py

```diff
@@ -86,28 +87,48 @@ def detection_score(self, y_true, y_pred):
 
 
 class OSPA(DetectionBaseScoreType):
+    """
+    Optimal Subpattern Assignment (OSPA) metric for IoU score.
+
+    This metric provides a coherent way to compute the miss-distance
+    between the detection and alignment of objects. Among all
+    combinations of true/predicted pairs, if finds the best alignment
+    to minimise the distance, and still takes into account missing
+    or in-excess predicted values through a cardinality score.
+
+    The lower the value the smaller the distance.
+
+    Arguments
+    ---------
+    name : str, optional
+        Method name
+    precision : int, optional
+        Rounding precision for the score (default is 2)
+    conf_threshold : float, optional
+        Confidence threshold value use for the Average Precision
+        measurement (default is 0.5)
+    minipatch : list of int, optional
+        Bounds of the internal scoring patch (default is None)
```

Comment:
Can you be more explitict here in the format ? (row_min, row_max, col_min, col_max ?)